www/widgets/DemoExampleMaker.js

```diff
@@ -291,6 +298,60 @@ class DemoExampleMaker extends Widget {
     this.$('[name="dem-url"]').style.display = 'block'
   }
 
+  _uploadJSON (dem) {
```

Comment:
lets use arrow function syntax `() => { }` so we can access `this` of the class && avoid having to pass this param in here